src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml

```diff
@@ -2,67 +2,269 @@
+                                </StackPanel>
+                                <StackPanel Grid.Column="2" Orientation="Horizontal">
+                                    <TextBlock x:Uid="MaxTextBlock"
+                                               Margin="0,4,0,0"
+                                               VerticalAlignment="Center"/>
+                                    <TextBox x:Name="MaxTextBox"
+                                             HorizontalAlignment="Left"
+                                             Style="{StaticResource ThemedVariableTextBoxStyle}"
+                                             GotFocus="TextBoxGotFocus"
+                                             KeyDown="TextBoxKeyDown"
+                                             LosingFocus="TextBoxLosingFocus"
+                                             Text="{x:Bind Max, Mode=OneWay}"/>
+                                </StackPanel>
+                            </Grid>
+                        </StackPanel>
+                    </Border>
+                </Grid>
+            </DataTemplate>
+
+        </ResourceDictionary>
+    </UserControl.Resources>
+    <Grid VerticalAlignment="Top">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="*"/>
+            <RowDefinition Height="Auto"/>
+        </Grid.RowDefinitions>
+        <mux:ItemsRepeaterScrollHost>
+            <ScrollViewer>
+                <StackPanel>
+                    <mux:ItemsRepeater x:Name="EquationInputList"
+                                       ItemsSource="{x:Bind Equations}"
+                                       TabFocusNavigation="Local">
+                        <mux:ItemsRepeater.ItemTemplate>
+                            <DataTemplate x:DataType="vm:EquationViewModel">
+                                <controls:EquationTextBox x:Uid="EquationInputButton"
+                                                          Margin="1,0,1,0"
+                                                          Style="{StaticResource EquationTextBoxStyle}"
+                                                          DataContext="{x:Bind Mode=OneWay}"
+                                                          EquationButtonClicked="EquationTextBox_EquationButtonClicked"
+                                                          EquationButtonContentIndex="{x:Bind FunctionLabelIndex, Mode=OneWay}"
+                                                          EquationColor="{x:Bind LineColor, Mode=OneWay}"
+                                                          EquationSubmitted="InputTextBox_Submitted"
+                                                          GotFocus="InputTextBox_GotFocus"
+                                                          HasError="{x:Bind GraphEquation.HasGraphError, Mode=OneWay}"
+                                                          IsAddEquationMode="{x:Bind IsLastItemInList, Mode=OneWay}"
```

Comment:
The last equation still has the remove button visible but clicking it does nothing, not sure if that is related to this branch.